akka-docs/rst/java/stream/stages-overview.rst

```diff
@@ -160,6 +160,25 @@ Emit each integer in a range, with an option to take bigger steps than 1.
 
 **completes** when the end of the range has been reached
 
+unfoldResource
+^^^^^
+Creates Source that works as dispatcher for legacy blocking IO API. There are three functions that pass as parameters.
+First one creates/opens resource on stream start. Second one reads next data element from resource each time demand message
+came from downstream. Third function closes/destroys resource.
```

Comment:
Formulated very differently than the other stage descriptions. What about something like "Wrap any resource that can be opened, blockingly queried for next element and closed using three distinct functions into a source.". I think the actual parameters is well enough described in the ScalaDoc/JavaDoc